components/base-war/src/main/webapp/resources/uicomponents/widgets/userpicker/userPicker.js

```diff
@@ -123,12 +123,12 @@ widgets.UserPicker = Class.create(widgets.Suggest, {
     var container = new Element('div', {'class': 'user'});
     var avatarWrapper = new Element('div', {'class': 'user-avatar-wrapper'});
     if (data.icon.indexOf('/') >= 0) {
-      avatarWrapper.insert(new Element('img', {src: data.icon, alt: data.info, 'class': 'icon'}));
+      avatarWrapper.insert(new Element('img', {src: data.icon, alt: data.info.escapeHTML(), 'class': 'icon'}));
```

Comment:
Unfortunately, there is a regression somewhere. Do not merge.
`(xwiki)` appears on the userpicker. Fixing...